bookomate/settings/base.py

```diff
@@ -96,7 +96,7 @@
 
 DATABASES = {
     'default': dj_database_url.config(
-        default=config('DATABASE_URL')
+        default="postgres://postgres:password@LOCALHOST:5432/Bookmate"
```

Comment:
This commit setting will break other devs setups. Please use the environment file as specified in the readme file.